src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -67,10 +68,11 @@ public static class TextCatalog
         /// Tokenize incoming text in input columns and output the tokens as output columns.
         /// </summary>
         /// <param name="catalog">The text-related transform's catalog.</param>
-        /// <param name="useMarkerCharacters">Whether to use marker characters to separate words.</param>
+        /// <param name="useMarkerCharacters">Whether to prepend a marker character, <see langword="0x02"/>, to the beginning,
+        /// and append another marker character, <see langword="0x03"/>, to the end of the output vector of characters.</param>
         /// <param name="columns">Pairs of columns to run the tokenization on.</param>
 
-        public static TokenizingByCharactersEstimator TokenizeCharacters(this TransformsCatalog.TextTransforms catalog,
+        public static TokenizingByCharactersEstimator ProduceCharactersAsKeys(this TransformsCatalog.TextTransforms catalog,
```

Comment:
>ProduceCharactersAsKeys [](start = 54, length = 23)

See my comment on the `TokenizeWords` extension. 
I would suggest `TokenizeCharactersAsKeys` as new name.